src/test/java/BaseTest.java

```diff
@@ -75,4 +101,21 @@ private static WebDriver pickBrowser(String browser) throws MalformedURLExceptio
         }
     }
 
+    @BeforeMethod
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/225230600-12b21e9a-a1cc-441a-8a44-d90a63a6ae56.png)

Reply:
The BaseTest on my end completely looks different. I do not have variables Threadlocal .... , parameters baseUrl is completely different on my end then it is in this picture. Something is wrong.
![Screenshot 2023-03-15 152554](https://user-images.githubusercontent.com/125710494/225421112-23084995-c160-481c-b73f-926b635ee849.png)

Reply:
Hello

 Please make changes, commit and push , then pull request and re-submit homework 23 for review